src/Symfony/Component/Intl/Countries.php

```diff
@@ -21,6 +21,8 @@
  */
 final class Countries extends ResourceBundle
 {
+    private static bool $optionalUserAssigned;
```

Comment:
I think the `$optionalUserAssigned` name is a bit problematic:

* `optional` feels redundant. If they are enabled explicitly by the user, it's clear that they are optional
* `user assigned` feels a bit misleading. Users (developers) are not assigning these territory/country codes. They are only enabling them to show them in the list of official countries.

Reply:
user-assigned is an ISO term, and somewhat internal detail (`private`)

optional is indeed a bit weird; maybe `$allowUserAssigned` is better